docs/installation/migrate.rst

```diff
@@ -16,77 +16,130 @@ You can set up continuous migration of |sr| to maintain a hybrid deployment ("ex
+
+* |ccloud| `5.2` or newer to serve as the destination |sr|
+* a local install of |cp| (for example, from a :ref:`ce-quickstart` download), or other cluster to serve as the origin |sr|
+
+Schema migration requires that you configure and run |crep-full|. If you need
+more information than is included in the examples here, refer to the
+:ref:`replicator tutorial` <replicator-quickstart>.
 
---------
 Run Book
 --------
 
 To migrate |sr| to |ccloud|, follow these steps:
 
-#.  Set the destination Schema Registry to IMPORT mode.  For example: 
+#.  Start the origin cluster.
 
+    If you are running a local cluster (for example, from a :ref:`ce-quickstart` download),
+    start only |sr| for the purposes of this tutorial.
+    
     .. code:: bash
+  
+        <path-to-confluent>/bin/confluent schema-registry
+    
+    Verify that `schema-registry`, `kafka`, and `zookeeper` are running and that
+    other services are down, as they can conflict with schema migration.
+    
+    ::
+    
+         control-center is [DOWN]
+         ksql-server is [DOWN]
+         connect is [DOWN]
+         kafka-rest is [DOWN]
+         schema-registry is [UP]
+         kafka is [UP]
+         zookeeper is [UP]
 
-        curl -X PUT -H "Content-Type: application/json" -H "Authorization: Basic dGVuYW50Mi1rZXk6bm9oYXNo" "http://destregistry:8081/mode" --data '{"mode": "IMPORT"}'
+#.  Set the destination |sr| to IMPORT mode.  For example: 
 
-#.  Configure Replicator with Schema Registry information.
+    .. code:: bash
+    
+        curl -u <schema-registry-api-key>:<schema-registry-api-secret> -X PUT -H "Content-Type: application/json" "http://destregistry:8081/mode" --data '{"mode": "IMPORT"}'
+
+#.  Configure a |crep| worker to specify the addresses of broker(s) in the destination cluster, as described in :ref:`config-and-run-replicator`.
+
+    The worker configuration file is in `<path-to-confluent>/etc/kafka/connect-standalone.properties`.
+
+    :: 
 
+    # Connect Standalone Worker configuration
+    bootstrap.servers=localhost:9092
+                
+#.  Configure :ref:`replicator` <replicator-quickstart>` with |sr| and destination cluster information.
+
+    For example, set the following in `etc/kafka-connect-replicator/quickstart-replicator.properties`:
 
     :: 
+
+        # destination cluster connection info
+        dest.kafka.ssl.endpoint.identification.algorithm=https
+        dest.kafka.sasl.mechanism=PLAIN
+        dest.kafka.request.timeout.ms=20000
+        dest.kafka.bootstrap.servers=<path-to-cloud-server>.cloud:9092
+        retry.backoff.ms=500
+        dest.kafka.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="2OSKHFEI22T7EJ7W" password="+UQUxhdL+R4hkQhzoUrtZg3FXwx/uLchTtXQRZ2NTGXd2zeQNeiBbaNe231tF7F+";
+        dest.kafka.security.protocol=SASL_SSL    
     
+        # Schema Registry migration topics to replicate from source to destination
+        topic.whitelist=_schemas
```

Comment:
1. @rayokota : why does `_schemas` need to be repeated in two different configuration parameters?  Why isn't `schema.topic` enough?

2. @londoncalling : users may believe (from previous experience on-prem to on-prem) that they can get away with just copying the `_schemas` topic.  Plus, due to #1 above, I think users may be even more confused.  Consider adding an explanation at the top of the page why this is insufficient.